lib/resources/r4/provenance.yaml

```diff
@@ -32,8 +32,7 @@ fields:
   note: |
     <ul>
       <li> Only one target may be provided.</li>
-      <li> The AllergyIntolerance, Condition, DocumentReference, Encounter, Immunization, MedicationRequest,
-        Observation, Patient and Procedure resources are supported.</li>
+      <li> The `AllergyIntolerance`, `CarePlan`, `CareTeam`, `Condition`, `DiagnosticReport`, `DocumentReference`,`Encounter`, `Goal`, `Immunization`, `MedicationRequest`, `Observation`, `Patient`, and `Procedure` resources are supported.</li>
```

Comment:
thoughts on this being a sub-list? if I'm looking to create Provenance for a resource, this is visually complex to find if it is supported.

Reply:
As with the markdown file, I'm making a similar change here.